### DISCUSSION TITLE
Select Component Optgroup/Category

### DISCUSSION THREAD

**Discussion by intrepidws** [filamentphp/filament#4019] [Help]
Is there a way to add an optgroup or category to the Select component options?

**Comment by ManojKiranA** [comment]
@danharrin any chance can we get this in v3 or do you already have it in roadmap ?

**Reply by danharrin** [reply]
Id rather keep this as a plugin, I already find it difficult to maintain the current Select component. Maybe this will change in the future.

**Reply by intrepidws** [reply]
Is there a plugin for this?

**Reply by danharrin** [reply]
https://filamentphp.com/plugins/simple-group-select
